canvg.js

```diff
@@ -2453,12 +2461,14 @@
 										var urlStart = srcs[s].indexOf('url');
 										var urlEnd = srcs[s].indexOf(')', urlStart);
 										var url = srcs[s].substr(urlStart + 5, urlEnd - urlStart - 6);
-										var doc = svg.parseXml(svg.ajax(url));
-										var fonts = doc.getElementsByTagName('font');
-										for (var f=0; f<fonts.length; f++) {
-											var font = svg.CreateElement(fonts[f]);
-											svg.Definitions[fontFamily] = font;
-										}
+										svg.parseXml(svg.ajax(url), function (image) {

```

Comment:
does `parseXml` need a callback param too? we're passing the callback a lot in the chain, maybe best to just store it on the svg object and call it if all images are loaded at the end